### PR TITLE
add version tags to docker images (closes #2315)

### LINKED ISSUE #2315: Version tags to docker images
### Are you requesting a feature or reporting a bug?

Enhancement

### What is the current behavior?

Only tags `latest`, `alpha` and `dev` for testcafe docker images

### What is the expected behavior?

In addition to current tags a unique version tag for each release. For example: `testcafe/testcafe:0.19.1`

### Reasoning

Without specific version tags our CI tests will fail when a non-backwards compatible image with `latest` tag is released. I'd like to pin a specific version to be used in CI.

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -752,16 +765,11 @@ gulp.task('docker-build', done => {
...
     }
 
-    const packageId = `${packageInfo.name}-${packageInfo.version}.tgz`;
-    const command = `docker build --no-cache --build-arg packageId=${packageId} -q -t testcafe -f docker/Dockerfile .`;
-    const imageId = childProcess.execSync(command, { env: process.env })
-        .toString()
-        .replace(/\n/g, '');
+    const packageId  = `${packageInfo.name}-${packageInfo.version}.tgz`;
+    const tagCommand = PUBLISH_TAGS.map(tag => `-t ${PUBLISH_REPO}:${tag}`).join(' ');
+    const command    = `docker build --no-cache --build-arg packageId=${packageId} -q ${tagCommand} -f docker/Dockerfile .`;
```
Comment (AndreyBelym) [inline Gulpfile.js]: Please, remove the `-q` switch here and in the `docker-test` [task](https://github.com/DevExpress/testcafe/pull/3111/files#diff-878989239e07228f7903fde98f450ed9R788) and add the `stdio: 'inherit'` option to the `execSync` call below.

**Comment by AlexKamaev** [comment]
@miherlosev ping

File: Gulpfile.js [hunk 2]
```diff
@@ -142,7 +142,20 @@ const CLIENT_TESTS_SAUCELABS_SETTINGS = {
 
 const CLIENT_TEST_LOCAL_BROWSERS_ALIASES = ['ie', 'edge', 'chrome', 'firefox', 'safari'];
 
-const PUBLISH_TAG = JSON.parse(fs.readFileSync(path.join(__dirname, '.publishrc')).toString()).publishTag;
+const PUBLISH_VERSION_RE = /^\d+\.\d+\.\d+(-alpha\.\d+)?$/;
+const PUBLISH_TAGS       = getPublishTags();
+const PUBLISH_REPO       = 'testcafe/testcafe';
+
+function getPublishTags () {
```
Comment (miherlosev) [inline Gulpfile.js]: Minor structural remark.
You call `getPublishTags` function before its declaration. 
I propose to change function signature as `function getPublishTags (packageInfo) {...}` and move this function to a separate file `docker/get-publish-tags`.

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Pull request merged**